sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+    constructor(endpoint: string, credential: KeyCredential, options?: DocumentAnalysisClientOptions);
+    // @internal
+    constructor(endpoint: string, credential: KeyCredential | TokenCredential, options?: DocumentAnalysisClientOptions);
+    beginAnalyzeDocuments(modelId: string, input: string | FormRecognizerRequestBody, options?: AnalyzeDocumentsOptions): Promise<AnalysisPoller>;
+    beginAnalyzeDocuments<Document>(model: DocumentModel<Document>, input: string | FormRecognizerRequestBody, options?: AnalyzeDocumentsOptions<AnalyzeResult<Document>>): Promise<AnalysisPoller<AnalyzeResult<Document>>>;
+    beginExtractGenericDocument(input: string | FormRecognizerRequestBody, options?: AnalyzeDocumentsOptions<GenericDocumentResult>): Promise<AnalysisPoller<GenericDocumentResult>>;
+    beginExtractLayout(input: string | FormRecognizerRequestBody, options?: AnalyzeDocumentsOptions<LayoutResult>): Promise<AnalysisPoller<LayoutResult>>;
+}
+
+// @public
+export interface DocumentAnalysisClientOptions extends CommonClientOptions {
+}
+
+// @public
+export interface DocumentAnalysisPollOperationState<Result = AnalyzeResult<AnalyzedDocument>> extends PollOperationState<Result> {
+    modelId: string;
+    operationId: string;
+    status: AnalyzeResultOperationStatus;
+}
+
+// @public
+export interface DocumentArrayField<T = DocumentField> extends DocumentFieldCommon {
+    // (undocumented)
+    kind: "array";
+    values: T[];
+}
+
+// @public
+export interface DocumentCountryRegionField extends DocumentFieldCommon {
+    // (undocumented)
+    kind: "countryRegion";
+    value?: string;
+}
+
+// @public
+export interface DocumentCurrencyField extends DocumentFieldCommon {
+    // (undocumented)
+    kind: "currency";
+    value?: string;
+}
+
+// @public
+export interface DocumentDateField extends DocumentValueField<Date> {
+    // (undocumented)
+    kind: "date";
+}
+
+// @public
+export interface DocumentEntity {
+    boundingRegions?: BoundingRegion[];
+    category: string;
+    confidence: number;
+    content: string;
+    spans: DocumentSpan[];
+    subCategory?: string;
+}
+
+// @public
+export type DocumentField = DocumentStringField | DocumentDateField | DocumentTimeField | DocumentPhoneNumberField | DocumentNumberField | DocumentIntegerField | DocumentSelectionMarkField | DocumentCountryRegionField | DocumentCurrencyField | DocumentSignatureField | DocumentArrayField | DocumentObjectField;
+
+// @public
+export interface DocumentFieldCommon {
+    boundingRegions?: BoundingRegion[];
+    confidence?: number;
+    content?: string;
+    spans?: DocumentSpan[];
+}
+
+// @public
+export interface DocumentFieldSchema {
+    description?: string;
+    example?: string;
+    items?: DocumentFieldSchema;
+    properties?: {
+        [propertyName: string]: DocumentFieldSchema;
+    };
+    type?: DocumentFieldType;
+}
+
+// @public
+export type DocumentFieldType = string;
```

Comment:
I think we decided before to not create aliases for primitive types. /cc @bterlson

Reply:
This is from the generated code.